code/datums/elements/mech_prying.dm

```diff
@@ -0,0 +1,72 @@
+/datum/element/mech_prying
```

Comment:
this element could have been an email

Reply:
if you have github email summaries on it will be